### PR TITLE
Update to install image, AJS 2.0 and help email

### PR DESCRIPTION
**Pull request by edwardupton**
### Proposed changes

Changes for clarity and new feature supporting AJS 2.0

### CODE REVIEW

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 1]
```diff
@@ -28,27 +28,27 @@ During the Segment connection setup, Littledata also adds a set of webhooks to y
...
+> This integration is maintained by Littledata _and is not supported by Segment directly_. The Littledata app has been reviewed by the Segment team for conformance with Segment's [E-Commerce Spec](/docs/connections/spec/ecommerce/v2/), and is the recommended way of using Segment with Shopify. However, it does require a paid subscription with Littledata, who mediates the connection between Shopify and Segment. [Contact the Littledata Support team](mailto:help@littledata.io) with any questions.
 
 ## Getting Started
 
 1. **Login** to your Shopify Store account.
 2. Go the [Shopify app store listing](https://apps.shopify.com/segment-com-by-littledata){:target="_blank"} for **_Segment.com by Littledata_**.
    ![](images/Nd5L0C6.png)
 3. Click **Add app** to begin the installation process.
-4. **Sign up** for a Littledata account using an email address, Google login or Facebook login. _More team members can be added to the subscription after completing the installation process._
+4. **Choose a Littledata subscription** suitable for your store's volume of monthly orders
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:46]:
```suggestion
4. **Choose a Littledata subscription** suitable for your store's volume of monthly orders.
```

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Littledata is available as an independent [Shopify App](https://apps.shopify.com
 
 During the [installation process](https://blog.littledata.io/help/posts/segment-installation-guide/), Littledata adds a `LittledataLayer.liquid` snippet to all pages (included in `theme.liquid`) on your Shopify store. The benefits of this approach include:
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:15]:
```suggestion
During the [installation process](https://blog.littledata.io/help/posts/segment-installation-guide/){:target="_blank"}, Littledata adds a `LittledataLayer.liquid` snippet to all pages (included in `theme.liquid`) on your Shopify store. The benefits of this approach include:
```

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 3]
```diff
@@ -28,27 +28,27 @@ During the Segment connection setup, Littledata also adds a set of webhooks to y
 - Customer data (e.g. email) securely relayed server-side
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:28]:
```suggestion
- Customer data (for example, email) securely relayed server-side
```

File: src/connections/sources/catalog/libraries/website/shopify-littledata/index.md [hunk 4]
```diff
@@ -28,27 +28,27 @@ During the Segment connection setup, Littledata also adds a set of webhooks to y
...
 
 Here's an architecture diagram that shows how the Littledata app mediates data flow between Shopify and Segment.
 
 ![](images/littledata_arch.png)
 
 > warning "Note"
-> This integration is maintained by Littledata _and is not supported by Segment directly_. The Littledata app has been reviewed by the Segment team for conformance with Segment's [E-Commerce Spec](/docs/connections/spec/ecommerce/v2/), and is the recommended way of using Segment with Shopify. However, it does require a paid subscription with Littledata, who mediates the connection between Shopify and Segment. [Contact the Littledata Support team](mailto:support@littledata.io) with any questions.
+> This integration is maintained by Littledata _and is not supported by Segment directly_. The Littledata app has been reviewed by the Segment team for conformance with Segment's [E-Commerce Spec](/docs/connections/spec/ecommerce/v2/), and is the recommended way of using Segment with Shopify. However, it does require a paid subscription with Littledata, who mediates the connection between Shopify and Segment. [Contact the Littledata Support team](mailto:help@littledata.io) with any questions.
```
Comment (rchinn-segment) [inline src/connections/sources/catalog/libraries/website/shopify-littledata/index.md:38]:
```suggestion
> This integration is maintained by Littledata _and isn't supported by Segment directly_. The Littledata app has been reviewed by the Segment team for conformance with Segment's [E-Commerce Spec](/docs/connections/spec/ecommerce/v2/), and is the recommended way of using Segment with Shopify. However, it does require a paid subscription with Littledata, who mediates the connection between Shopify and Segment. [Contact the Littledata Support team](mailto:help@littledata.io) with any questions.
```

**Review by rchinn-segment — CHANGES_REQUESTED** [review]
Looks good @edwardupton. Just a few requested changes.

**Comment by edwardupton** [comment]
@rchinn-segment all resolved

**Review by rchinn-segment — APPROVED** [review]

**Pull request merged**